builtin/logical/pki/path_acme_new_account.go

```diff
@@ -28,6 +28,24 @@ func pathAcmeIssuerAndRoleNewAccount(b *backend) *framework.Path {
 			"/roles/"+framework.GenericNameRegex("role")+"/acme/new-account")
 }
 
+func pathAcmeRootUpdateAccount(b *backend) *framework.Path {
+	return patternAcmeNewAccount(b, "acme/account/"+framework.MatchAllRegex("kid"))
+}
+
+func pathAcmeRoleUpdateAccount(b *backend) *framework.Path {
+	return patternAcmeNewAccount(b, "roles/"+framework.GenericNameRegex("role")+"/acme/account/"+framework.MatchAllRegex("kid"))
+}
+
+func pathAcmeIssuerUpdateAccount(b *backend) *framework.Path {
+	return patternAcmeNewAccount(b, "issuer/"+framework.GenericNameRegex(issuerRefParam)+"/acme/account/"+framework.MatchAllRegex("kid"))
```

Comment:
Do you need to include `kid` in the `patternAcmeNewAccount` matcher?

Reply:
Yup, we will need it for the other change later on when `kid` isn't the hash of the public key

Reply:
Added kid to just the account update API definitions.